store/sqlstore/team_store.go

```diff
@@ -1006,20 +983,7 @@ func (s SqlTeamStore) GetAllForExportAfter(limit int, afterId string) ([]*model.
 }
 
 // GetUserTeamIds get the team ids to which the user belongs to
-func (s SqlTeamStore) GetUserTeamIds(userID string, allowFromCache bool) ([]string, *model.AppError) {
-	if allowFromCache {
-		if cacheItem, ok := allTeamIdsForUserCache.Get(userID); ok {
-			if s.metrics != nil {
-				s.metrics.IncrementMemCacheHitCounter("All Team Ids for User")
-			}
-			return cacheItem.([]string), nil
-		}
-	}
-
-	if s.metrics != nil {
-		s.metrics.IncrementMemCacheMissCounter("All Team Ids for User")
-	}
-
+func (s SqlTeamStore) GetUserTeamIds(userID string, _ bool) ([]string, *model.AppError) {
```

Comment:
I would leave the `allowFromCache` param as it was. Was some [linting](https://en.wikipedia.org/wiki/Lint_(software)) tool complaining over this?

Reply:
No, it wasn't a linting tool but the `allowFromCache` should not be used (now) inside of any store, but in the cache layer, so in order to prevent its usage I left it that way

Reply:
Alright, it's just that seeing an empty param doesn't give the developer any clue of what's going on. Probably a comment documenting the fact that the param will be deprecated and not used anymore would be the way to go.